docroot/profiles/custom/cgov_site/modules/custom/cgov_schema_org/src/Plugin/Field/FieldWidget/CgovSchemaOrgWidget.php

```diff
@@ -48,6 +48,7 @@ public function formElement(
       'faq_page',
     ];
     $item = $items[$delta];
+    /* @phpstan-ignore-next-line */
```

Comment:
Fix this to not need ignore. This seems like $item->get('page_itemtype')->value or something...

Reply:
Resolved.